modules/migration-mtc-cr-manifests.adoc

```diff
@@ -0,0 +1,363 @@
+// Module included in the following assemblies:
+//
+// * migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc
+// * migration/migrating_4_1_4/migrating-applications-with-cam-4-1-4.adoc
+// * migration/migrating_4_2_4/migrating-applications-with-cam-4-2-4.adoc
+
+[id="migration-mtc-cr-manifests_{context}"]
+= {mtc-short} custom resource manifests
+
+{mtc-full} ({mtc-short}) uses the following custom resource (CR) manifests to create CRs for migrating applications.
+
+[id="directimagemigration_{context}"]
+== `DirectImageMigration`
+
+`DirectImageMigration` copies images directly from the source cluster to the destination cluster.
+
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: DirectImageMigration
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: <directimagemigration_name>
+spec:
+  srcMigClusterRef:
+    name: <source_cluster_ref> <1>
+    namespace: openshift-migration
+  destMigClusterRef:
+    name: <destination_cluster_ref> <2>
+    namespace: openshift-migration
+  namespaces:
+  - <namespace> <3>
+----
+<1> Specify the `MigCluster` CR name of the source cluster.
+<2> Specify the `MigCluster` CR name of the destination cluster.
+<3> Specify one or more namespaces containing images to be migrated.
+
+[id="directimagestreammigration_{context}"]
+== `DirectImageStreamMigration`
+
+`DirectImageStreamMigration` copies image stream references directly from the source cluster to the destination cluster.
```

Comment:
s/`DirectImageStreamMigration`/A direct image stream migration/

Reply:
I prefer "The `DirectImageStreamMigration` CR ..." because this is not a concept module. It is a reference module.